deployments/ocean/image/binder/dask_config.yaml

```diff
@@ -21,18 +21,18 @@ kubernetes:
           - '2'
           - --no-bokeh
           - --memory-limit
-          - 12GB
+          - "11.5GB"
           - --death-timeout
           - '60'
         image: ${JUPYTER_IMAGE_SPEC}
         name: dask-${JUPYTERHUB_USER}
         resources:
           limits:
             cpu: "1.75"
-            memory: 12G
+            memory: "11.5G"
           requests:
-            cpu: 1
-            memory: 12G
+            cpu: "1.75"
```

Comment:
I think it would make sense to lower this, as if you have two pods on a node and one is inactive currently, the other could if needed get more power by not making the pod hog a fixed amount that they wont share.

I also think this will be an optimization without drawbacks, and perhaps no real benefit either depending on the kind of work that is done by the worker.